### DISCUSSION TITLE
Problem with my addon

### DISCUSSION THREAD

**Discussion by OmarMohamedAero** [freeplane/freeplane#1888] [Help]
I'm creating a new addon and I need help. I've written the script and followed the steps, but I'm getting an error message when I try to publish it. Here's a picture of the error message

![image](https://github.com/freeplane/freeplane/assets/78862367/f57ee574-a959-4c0b-911a-4f334448bba8)

**Comment by HaaiHenkie** [comment]
Hello Omar Ahmed,

You need to add your add-on's homepage as a hyperlink to the root node of your add-on's build map with Ctrl+K. The note of the root node describes anything you should or could add to the root node.

Best regards,

Henk van den Akker

**Reply by OmarMohamedAero** [reply]
Thanks, i'm trying

**Comment by macmarrum** [comment]
You can see what's needed by examining an existing add-on, e.g. Pack and Go-Go. Download https://github.com/macmarrum/freeplane-PackAndGoGo/releases/download/v2024.3.19/PackAndGoGo-v2024.3.19.addon.mm and open it in Freeplane but choose not to install it when prompted.

**Reply by OmarMohamedAero** [reply]
Thanks

**Reply by OmarMohamedAero** [reply]
Working on it
